docs/cosmos-finality-provider-operation.md

```diff
@@ -0,0 +1,821 @@
+      2. [Withdraw Rewards](#522-withdraw-rewards)
+      3. [Set Withdraw Address](#523-set-withdraw-address)
+   3. [Start Finality Provider](#53-start-finality-provider)
+   4. [Status of Finality Provider](#54-status-of-finality-provider)
+   5. [Edit finality provider](#55-edit-finality-provider)
+   6. [Jailing and Unjailing](#56-jailing-and-unjailing)
+   7. [Slashing and Anti-slashing](#57-slashing-and-anti-slashing)
+   8. [Prometheus Metrics](#58-prometheus-metrics)
+6. [Recovery and Backup](#6-recovery-and-backup)
+   1. [Critical Assets](#61-critical-assets)
+   2. [Backup Recommendations](#62-backup-recommendations)
+   3. [Recover finality-provider db](#63-recover-finality-provider-db)
+      1. [Recover local status of a finality provider](#631-recover-local-status-of-a-finality-provider)
+      2. [Recover public randomness proof](#632-recover-public-randomness-proof)
+
+
+## 1. Prerequisites
+
+Before proceeding with this guide, you must complete the EOTS daemon setup 
+process described in [EOTS Daemon Setup](./eots-daemon.md). This includes:
+
+* Installing the finality provider toolset (`cosmos-fpd` and `eotsd` binaries)
+* Initializing and configuring the EOTS daemon
+* Adding your EOTS key to the daemon
+* Starting the EOTS daemon service
+
+⚠️ **Important considerations:**
+* The EOTS daemon must be running and accessible before you can operate a 
+  finality provider
+* Each Finality Provider must generate a new EOTS key - EOTS keys cannot be 
+  reused across different finality providers
+
+## 2. System Requirements
+
+Recommended specifications for running a Finality Provider:
+
+* CPU: 2 vCPUs
+* RAM: 4GB
+* Storage: 50GB SSD/NVMe
+* Network: Stable internet connection
+* Security:
+  * Encrypted storage for keys and sensitive data
+  * Regular system backups
+
+These are the minimum specifications for running a finality provider.
+Requirements may vary based on network activity and your operational needs.
+For production environments, you may want to consider using more robust hardware.
+
+For security tips of running a finality provider, please refer to
+[Slashing Protection](./slashing-protection.md),
+[HMAC Security](./hmac-security.md),
+and [Recovery and Backup](#6-recovery-and-backup).
+
+## 3. Keys Involved in Finality Provider Operation
+
+Operating a finality provider involves managing multiple keys, each serving distinct 
+purposes. Understanding these keys, their relationships, and security implications is 
+crucial for secure operation.
+
+| Aspect | EOTS Key                                                                                                                                                            | Babylon Genesis Key                                                                                                                                             | Cosmos BSN Key                                                                                 |
```

Comment:
Should we call this Operation Key as in other docs, or Cosmos vs Consumer BSN key? cc @vitsalis